.github/workflows/s3-snapshots.yml

```diff
@@ -107,44 +68,30 @@ jobs:
           aws-region: eu-central-1
 
       - name: Copy files to dokka's S3 bucket
-        run: |
-          aws s3 --recursive rm s3://${{ env.bucket-name }}/${{ env.branch-name }}/stdlib/prev
-          aws s3 --recursive mv s3://${{ env.bucket-name }}/${{ env.branch-name }}/stdlib/latest s3://${{ env.bucket-name }}/${{ env.branch-name }}/stdlib/prev
-          aws s3 sync ./stdlib/build/dokka s3://${{ env.bucket-name }}/${{ env.branch-name }}/stdlib/latest
+        run: ./integration-tests/aws_sync.sh s3://${{ env.bucket-name }} stdlib ../stdlib
 
       - name: Print link
-        run: echo http://dokka-snapshots.s3.eu-central-1.amazonaws.com/${{ env.branch-name }}/stdlib/latest/kotlin-stdlib/kotlin-stdlib/index.html
+        run: echo http://dokka-snapshots.s3.eu-central-1.amazonaws.com/${{ env.branch-name }}/stdlib/${GITHUB_SHA::7}/index.html
 
-  biojava:
```

Comment:
can we keep bio java? if we dont have a way to test it using an integration test at least keep it in the snapshots

Reply:
Yup, readded biojava